Source/WebCore/platform/graphics/cocoa/VP9UtilitiesCocoa.mm

```diff
@@ -155,23 +145,12 @@ bool isVP9DecoderAvailable()
         return true;
 
 #if PLATFORM(IOS) || PLATFORM(VISION)
-    return canLoad_VideoToolbox_VTIsHardwareDecodeSupported() && VTIsHardwareDecodeSupported(kCMVideoCodecType_VP9);
+    return vp9HardwareDecoderAvailable();
 #else
-    if (!VideoToolboxLibrary(true))
-        return false;
-    return noErr == VTSelectAndCreateVideoDecoderInstance(kCMVideoCodecType_VP9, kCFAllocatorDefault, nullptr, nullptr);
+    return true;
 #endif
 }
 
-bool isVP8DecoderAvailable()
-{
-    if (isSWDecodersAlwaysEnabled())
-        return true;
-    if (!VideoToolboxLibrary(true))
-        return false;
-    return noErr == VTSelectAndCreateVideoDecoderInstance('vp08', kCFAllocatorDefault, nullptr, nullptr);
```

Comment:
Ditto, we need to check that libvpx/libwebrtc is available, WebRTCProvider::webRTCAvailable.

Reply:
fine